src/library/scala/collection/mutable/Buffer.scala

```diff
@@ -227,5 +227,4 @@ object Buffer extends SeqFactory.Delegate[Buffer](ArrayBuffer)
 object IndexedBuffer extends SeqFactory.Delegate[IndexedBuffer](ArrayBuffer)
 
 /** Explicit instantiation of the `Buffer` trait to reduce class file size in subclasses. */
-@SerialVersionUID(3L)
```

Comment:
I remember this lint. It was meant to victim-blame the victims of some SVUID bug I introduced, I think.